bokeh/charts/builders/bar_builder.py

```diff
@@ -60,7 +61,7 @@ class BarBuilder(Builder):
     values = Dimension('values')
 
     dimensions = ['values']
-    #req_dimensions = [['values']]
+    # req_dimensions = [['values']]
```

Comment:
is there any reason we should keep this commented code here? If so we should add more context..